.env.example

```diff
@@ -36,7 +36,7 @@ NORAY_UDP_RELAY_TIMEOUT=30s
 NORAY_UDP_RELAY_CLEANUP_INTERVAL=30s
 
 # Port where noray listens for UDP relay requests from hosts
-NORAY_UDP_REGISTRAR_PORT=8809
+NORAY_UDP_REGISTRAR_PORT=8890
```

Comment:
Why this change? The default registrar port is 8809 as can be seen [here](https://github.com/foxssake/noray/blob/0db49adb4be3fcbebcfb885d7823fd56279626cb/src/config.mjs#L38).

Reply:
The idea behind the change was to use the same port number as TCP to avoid having varying protocol ports open across different ports. Made it for me just easier to be aware of which ports are actually open and not feel like "wasting" any ports. I didn't get too deep into the code when I made that change to see there is a default option set, in hindsight that makes a lot of sense.